src/components/SearchBox.js

```diff
@@ -17,55 +16,58 @@ class SearchBox extends Component {
     this.handleDeleteAll = this.handleDeleteAll.bind(this);
     this.handleGoBack = this.handleGoBack.bind(this);
   }
-
+  results = [];
   handleGoogle() {
     window.open("http://www.google.com/search?q=" + this.state.value, "_blank");
     console.log("googling");
   }
   handleChange(e) {
     this.setState({ value: e.target.value });
   }
-  componentDidMount() {
```

Comment:
This is where we were previously creating an iframe, provided by Google, each time the page reloaded & the component mounted again